.idea/codestyles/Project.xml

```diff
@@ -1,35 +1,11 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <JavaCodeStyleSettings>
```

Comment:
This is just an updated version of the file exported from Android Studio

Reply:
Ideally this would be in a separate PR but no worries to include it here